mssql-plugin/src/e2e-test/resources/pluginParameters.properties

```diff
@@ -40,8 +40,8 @@ uniqueIdentifierColumnsList=(ID, COL1)
 uniqueIdentifierValues=VALUES ('User1', '6F9619FF-8B86-D011-B42D-00C04FC964FF')
 outputDatatypesSchema3=[{"key":"ID","value":"string"},{"key":"COL1","value":"string"}]
 
-dateTimeColumns=(ID VARCHAR(100) PRIMARY KEY, COL1 DATETIME, COL2 DATETIME2(0))
-dateTimeColumnsList=(ID, COL1, COL2)
-dateTimeValues=VALUES ('User1', '2023-01-01 01:00:00.000', '2023-01-01 01:00:00.000')
+dateTimeColumns=(ID VARCHAR(100) PRIMARY KEY, COL1 DATETIME, COL2 DATETIME2(0), COL3 SMALLDATETIME)
+dateTimeColumnsList=(ID, COL1, COL2, COL3)
+dateTimeValues=VALUES ('User1', '2023-01-01 01:00:00.000', '2023-01-01 01:00:00.000', '2023-01-01 01:00:00.000')
 outputDatatypesSchema4=[{"key":"ID","value":"string"},{"key":"COL1","value":"datetime"},\
-  {"key":"COL2","value":"datetime"}]
+  {"key":"COL2","value":"datetime"},{"key":"COL3","value":"datetime"}]
```

Comment:
Please add a test for DatetimeOffset type as well.

Reply:
added